stdlib/public/TensorFlow/ShapedArray.swift

```diff
@@ -578,6 +578,27 @@ extension ShapedArray : CustomReflectable {
   }
 }
 
+/// Codable conformance.
+extension ShapedArray : Codable where Scalar : Codable {
+  private enum CodingKeys: String, CodingKey {
```

Comment:
Currently, almost all `CodingKeys` types in stdlib SDKs conform to `Int`, not `String`:
```console
$ grep -nr "enum CodingKeys"
public/SDK/MediaPlayer/MPMusicPlayerPlayParameters.swift:64:    internal enum CodingKeys: String, CodingKey {
public/SDK/Foundation/URLComponents.swift:476:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/PersonNameComponents.swift:150:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/IndexSet.swift:859:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/IndexPath.swift:798:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/Calendar.swift:1132:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/DateComponents.swift:356:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/URL.swift:1221:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/TimeZone.swift:281:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/CharacterSet.swift:811:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/Measurement.swift:282:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/Locale.swift:485:    private enum CodingKeys : Int, CodingKey {
public/SDK/Foundation/Decimal.swift:473:    private enum CodingKeys : Int, CodingKey {
```
Here's what the difference looks like for `ShapedArray`:
```swift
// enum CodingKeys : String
// {"shape":[2,3],"scalars":[1,2,3,4,5,6]}
// enum CodingKeys : Int
// {0:[2,3],1:[1,2,3,4,5,6]}
```